Framework/Core/Windows/Devices/I2c/I2cEnums.cs

```diff
@@ -0,0 +1,21 @@
+namespace Windows.Devices.I2c
+{
+    public enum I2cBusSpeed
+    {
+        StandardMode = 0,   // 100kHz

```

Comment:
I am just curious to know why you always set the integer value of the first member of an enum to zero. Isn't zero always the default integer value for the first member of any enum?